draco/analysis/powerspectrum.py

```diff
@@ -69,13 +71,110 @@ def process(self, klmodes):
 
         q_list = []
 
+        st = time.time()
+
         for mi, m in klmodes.vis[:].enumerate(axis=0):
             ps_single = pse.q_estimator(m, klmodes.vis[m, : klmodes.nmode[m]])
             q_list.append(ps_single)
 
         q = klmodes.comm.allgather(np.array(q_list).sum(axis=0))
         q = np.array(q).sum(axis=0)
 
+        et = time.time()
+        if mpiutil.rank0:
+            print("m, time needed for quadratic pse", m, et - st)
```

Comment:
This looks like left over debug statements. Either remove it, or at least improve the formatting.

Reply:
Would be good to keep, but agree about improving formatting